chirp/drivers/mml_jc8810.py

```diff
@@ -205,6 +205,8 @@
 PTTLT_LIST = ["None", "100 ms"] + \
              ["%s ms" % x for x in range(200, 1200, 200)]
 QTSAVE_LIST = ["All", "RX", "TX"]
+ROGER_LIST = ["OFF", "BEEP", "TONE1200"]
+ROGER470_LIST = ["OFF", "BEEP", "TONE1200 [fw v1.22+]"]
```

Comment:
I would really rather we not do this (make an option with a firmware version in it), especially since it's not like the user needs to select one or the other. It looks to me like below you're able to determine which model by looking at `MODEL`, why do we need different lists with slightly different names? Can't you do the right thing for the newer firmware if `MODEL==RT470L` in the set without the user having to see the difference?

Reply:
I would really rather not do this either. But I can't help the fact that the RT-470 previous choices were ON/OFF (which I handled using RadioSettingValueBoolean()) and  (currently latest released) fw v1.22 changed them to OFF/BEEP/TONE1200.

The RT-470L is not an RT-470 with newer firmware. It is a totally different radio. It has different rf.valid_power_levels, different rf.valid_bands and different supported menus.

The RT-470 user has to "see" the difference because if the radio has not had its firmware upgraded to v1.22, then how will the user know that the choosing 'TONE1200' will do nothing (most likely behave the same as selecting OFF)? Not knowing creates the potential for the user to open a bug report because choosing 'TONE1200' didn't result in the radio emitting 'TONE1200' when the PTT button is released. 

It is the same situation for the TOP KEY (Short Press) selection. The RT-470 fw v1.22 added 'hidden' support for the TOP KEY (Short Press). If the CHIRP user with an RT-470 having a firmware version prior to v1.22 attempts to make any changes to the TOP KEY (Short Press) setting in CHIRP, nothing will happen in the radio. The TOP KEY (Short Press) is not supported prior to v1.22 so short pressing the Top Key does nothing.

It would be great if the RT-470 image contained the firmware version. I would have used this to cleanly display what settings that use had available. But I have searched the RT-470 image (even after expanding it to 64k) for anything that resembles the firmware version and have not been able to locate anything. I have also reached out to some RT-470 owners that have been actively hacking the radio to see if they have discovered any bytes that would seem to reveal the firmware version and so far nothing.

My only other solution would be to have the firmware version in the "MODEL =" that would allow to choose the RT-470 model based on the radio's firmware version. I don't consider this an acceptable solution.

So what I have done, and should be pushing later today, is to rip out all the "v1.22" options from the driver. Everything for the RT-470 is now based on the radio having v1.22 firmware. I have added "Notice to Developer" comments that state which firmware version that the driver support for each model is based on. I have also added comments at the setting that mention what the v1.22 firmware added/changed and that these additions/changes are not compatible with previous RT-470 firmware versions.